Automattic/sensei
# Display warning in the editor when adding the lesson properties block if Learning Mode is enabled

Fixes #6016

### The problem

- Currently, the way we have to add the Lesson Properties to a lesson is through the Lesson Properties block.
- We have a Lesson Properties block to be used in the Learning Mode templates, but to use this, we need to set the lesson properties in the lessons.
- Before this PR, we'd render the Lesson Properties in the editor, but we wouldn't render it in the frontend, causing some confusion.

### Changes proposed in this Pull Request

* This PR just changes the editor rendering in the lesson to explain to the user that this block won't render in the frontend when Learning Mode is activated.

### Testing instructions


* Create a lesson with Learning Mode enabled.
* Add a Lesson Properties block.
* Make sure the block display a warning in the editor.
* Disable Learning Mode.
* Return to the editor, and make sure now the Lesson Properties block is displayed without any warning.
